cli/build.rs

```diff
@@ -375,6 +378,10 @@ fn main() {
     }
   }
 
+  // Linux + aarch64 does not support a glibc version that supports `--export-dynamic-symbol`.
+  #[cfg(all(target_os = "linux", target_arch = "aarch64"))]
+  println!("cargo:rustc-link-arg-bin=deno=-rdynamic");
```

Comment:
GNU aarch64 ld did not release 2.38? That's weird 🤔

Reply:
https://man.archlinux.org/man/aarch64-linux-gnu-ld.1.en#export~2 It did. Can you upgrade your local binutils to >=2.38 and revert this back to `--export-dynamic-symbol`? `-rdynamic` will drastically increase the binary size (>160MB)

Reply:
The issue is that the compiled binary is used in OSes with old versions of glibc. 

I’m not very knowledgeable about this, but I had a hard time upgrading binutils without also depending on a newer glibc. 

Hypothetically, if I upgraded just binutils to >= 2.38, could I leave glibc at 2.27 (the version that ships with Ubuntu 18.04). Seems like a headache. 

Suggestion: can I update build.rs to check on the glibc version is >= 2.28, and de-optimise the build in that case, maybe with a warning?)

Reply:
> can I update build.rs to check on the glibc version is >= 2.28, and de-optimise the build in that case, maybe with a warning? 

Yup that sounds great. It can be done in a follow up. I'll merge this PR as-is